storage/replica.go

```diff
@@ -2497,9 +2501,9 @@ func (r *Replica) maybeGossipSystemConfig() {
 	r.systemDBHash = hash
 }
 
-// newReplicaCorruptionError creates a new error indicating a corrupt replica,
+// NewReplicaCorruptionError creates a new error indicating a corrupt replica,
 // with the supplied list of errors given as history.
-func newReplicaCorruptionError(errs ...error) *roachpb.ReplicaCorruptionError {
+func NewReplicaCorruptionError(errs ...error) *roachpb.ReplicaCorruptionError {
 	var errMsg string
 	for i := range errs {
 		err := errs[len(errs)-i-1]
```

Comment:
not your code, but we shouldn't ever need to pass multiple errors in here and do this dance because we now have `errors.Wrap(err, "cause")` which should be used at the callsite (and we would simply wrap again here).